app/controllers/transactions_controller.rb

```diff
@@ -81,6 +83,10 @@ def transaction_params
 
   def set_transaction
     @transaction = Transaction.find(params[:id])
+
+  rescue ActiveRecord::RecordNotFound
+    redirect_to transactions_path,
+      alert: "Transaction #{transaction_url.split("/").last} was deleted or might never existed"
```

Comment:
Simply call `routing_error`

Reply:
Parang magkaiba po kasi sila e

https://user-images.githubusercontent.com/10540193/135968966-08d1b613-8f23-41bd-94a4-c025f513b43b.mov

Reply:
Not found is 404.  Doesn't matter if the user is deleting, editing, just changing the id to a high number.